src/styles/global.css

```
@@ -78,6 +89,93 @@ a {
 	margin-left: 0.2em;
 }
 
+.motion-enter {
+	--motion-delay: 0ms;
+	opacity: 0;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)